src/prefect/agent.py

```diff
@@ -25,7 +25,7 @@ class OrionAgent:
     def __init__(
         self,
         work_queues: List[str] = None,
-        work_queue_prefix: str = None,
+        work_queue_prefix: List[str] = None,
```

Comment:
This and the change in the client are technically breaking changes as people can be using these interfaces. Can you update this to take a `Union[str, List[str]]` and just coerce the `str` to ` List[str]` in `__init__`?

Reply:
Done.